main.py

```diff
@@ -1 +1,118 @@
 
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+import time
+
+class Testcase101:
+
+    def main(self):
+        driver = webdriver.Firefox(executable_path="C:\\Users\\Johny\\Downloads\\geckodriver-v0.33.0-win64\\geckodriver.exe")
+        driver.get("https://interview.supporthive.com/staff/")
+        driver.implicitly_wait(30)
```

Comment:
Instead of Imlicit wait we can use expicit wait for better control over waiting conditions.